Resources/Locales/ko/openspades.po

```diff
@@ -1074,12 +1073,12 @@ msgid ""
 "FXAA: Performs antialiasing by smoothing artifacts out as a post-process.|"
 "Off|MSAA 2x|4x|FXAA"
 msgstr ""
-"앤티앨리어싱:오브젝트의 윤곽에 발생하는 계단 현상을 줄여 줍니다.\n"
+"앤티앨리어싱: 오브젝트의 윤곽에 발생하는 계단 현상을 줄여 줍니다.\n"
```

Comment:
Please remove the space after the first colon (please see the source text). The first colon here is interpreted as a separator between the item title and description.